server/connectors/slack_connector/client.py

```
@@ -3,15 +3,20 @@
 Uses the stored access_token from OAuth to interact with Slack workspace.
 """
 
+import asyncio
 import logging
 import time
 import requests
-from typing import Dict, Any, List, Optional
+from typing import Dict, Any, Iterator, List, Optional
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)